consoles/sshVirtsh.pm

```diff
@@ -229,6 +229,17 @@ sub add_pty ($self, $args) {
     return;
 }
 
+sub add_usb_hub ($self) {
+    my $doc = $self->{domainxml};
+    my $devices = $self->{devices_element};
+
+    my $controller = $doc->createElement('controller');
+    $controller->setAttribute(type => 'usb');
+    $controller->setAttribute(index => '0');
```

Comment:
What happens when this method is called multiple times? Would it work? If not, perhaps it would be a consideration to make it safe to call multiple times by storing this index as variable and incrementing it each time.

Reply:
xen PV VM start to fail when it is called twice. I would leave it as this since we dont need multiple usb hubs and the parameters can be added later

```
      </disk>
      <controller type='usb' index='0' ports='8'/>
      <controller type='usb' index='0' ports='8'/>
      <controller type='xenbus' index='0'/>
      <interface type='network'>
```